dnf.spec

```diff
@@ -147,13 +154,12 @@ Common data and configuration files for DNF
 %package -n %{yum_subpackage_name}
 Requires:       %{name} = %{version}-%{release}
 Summary:        %{pkg_summary}
-%if 0%{?fedora}
 %if 0%{?fedora} >= 31
-Conflicts:      yum
+Provides:       %{name}-yum = %{version}-%{release}
+Obsoletes:      %{name}-yum < 4.2.1
```

Comment:
This will not work in future. The next release into Fedora 30 will break it. What about ```Obsoletes:      %{name}-yum < %{version}-%{release}```?